h/accounts/models.py

```diff
@@ -130,6 +130,11 @@ def is_activated(self):
     _password = sa.Column('password', sa.UnicodeText(), nullable=False)
     # Password salt
     salt = sa.Column(sa.UnicodeText(), nullable=False)
+    # Last password update
+    password_updated = sa.Column(sa.DateTime(),
+                                 default=sa.sql.func.now(),
```

Comment:
`sa.sql.func` == `sa.func`, so the difference between the `default` and `server_default` lines is a bit confusing here.